zfs-prune-snapshots

```diff
@@ -184,8 +192,15 @@ while getopts 'hniqlRp:s:vV' option; do
 		p) prefix=$OPTARG;;
 		s) suffix=$OPTARG;;
 		q) quiet=true; exec 1>/dev/null;;
-		R) recursive=true;;
-		v) ((verbosity++));;
+		e) error_exit=true;;
+		r) recursive=true
```

Comment:
I see that specifying `-rR` or `-Rr` will result in the last option that is set "winning"... would it make more sense to just error if the user specifies both `-r` and `-R`?

For example, have a block of code below the argument parsing like:

``` bash
if $recursive && $recursive_extended; then
  echo 'cannot specify -r and -R together' >&2
  exit 1
fi
```